templates/site/js/catalog-search.js

```diff
@@ -143,3 +157,1170 @@ function addDefaultSearchAttributes(inputBox){
+    if(searchResultDiv.innerHTML != "")
+    {
+        searchResultDiv.innerHTML = "";
+    }
+
+    var searchResultTable = document.createElement("table");
+    var tableHeader = document.createElement("thead");
+    var tableTitleFormatCell = document.createElement("td");
+    var tableTitleDatasetCell = document.createElement("td");
+    var tableTitleStartDatetimeCell = document.createElement("td");
+    var tableTitleEndDatetimeCell = document.createElement("td");
+    var errorRow = document.createElement("tr");
+    var errorCell = document.createElement("td");
+    var errorMessage = document.createElement("p");
+
+    errorMessage.classList.add("subtitle-1", "error-search-results");
+
+    tableTitleFormatCell.innerText = "Format";
+    tableTitleDatasetCell.innerText = "Dataset ID";
+    tableTitleStartDatetimeCell.innerText = "Start Datetime";
+    tableTitleEndDatetimeCell.innerText = "End Datetime";
+    errorMessage.innerText = "No results with the current query";
+
+    errorCell.appendChild(errorMessage);
+    errorRow.appendChild(errorCell);
+
+    tableHeader.appendChild(tableTitleDatasetCell);
+    tableHeader.appendChild(tableTitleStartDatetimeCell);
+    tableHeader.appendChild(tableTitleEndDatetimeCell);
+    tableHeader.appendChild(tableTitleFormatCell);
+    var tableBody = document.createElement("tbody");
+    var assetType;
+
+
+    searchResultTable.appendChild(tableHeader);
+    searchResultTable.appendChild(tableBody);
+    searchResultDiv.appendChild(searchResultTable);
+
+
+    if(Object.entries(json.features).length > 0) {
+        Object.entries(json.features).forEach(([featureKey, featureValue]) => {
+            var rowSearchResult = document.createElement("tr");
+            tableBody.appendChild(rowSearchResult);
+
+            var collectionAnchor = document.createElement("a");
+            var cellFormat = document.createElement("td");
+            cellFormat.classList.add("search-results-format");
+
+            if (featureValue.id) {
+                var cellDatasetTitle = document.createElement("td");
+                var linkDatasetTitle = document.createElement("a");
+                var datasetTitleArray;
+                linkDatasetTitle.setAttribute("role", "button");
+
+                linkDatasetTitle.onclick = function () {
+                    swapDatasetDetails();
+                    swapBackButtonText();
+                    populateDatasetDetails(featureValue);
+                };
+
+                linkDatasetTitle.innerText = featureValue.id;
+
+                cellDatasetTitle.appendChild(linkDatasetTitle);
+                rowSearchResult.appendChild(cellDatasetTitle);
+            }
+
+
+            if (featureValue.properties.start_datetime) {
+                var cellStartDateTimeValue = document.createElement("td");
+
+                cellStartDateTimeValue.innerText = new Date(featureValue.properties.start_datetime);
```

Comment:
actually format the date, don't just rely on the default string representation because then you get inconsistent timezone formats:

![image](https://github.com/user-attachments/assets/a0026da1-0f24-4062-aad3-1a353c4e00ae)